tests/behavioural/features/message_get.feature

```diff
@@ -41,3 +41,9 @@ Feature: Message get by ID Endpoint
     Given an external user has created and saved a draft message
     When the external user navigates to draft messages and opens the draft message
     Then the draft message should be displayed
+
+  @ignore
+  Scenario: As a user when I send a invalid message with bad syntax
+    Given a user is sending a message
```

Comment:
Also we should have a test for each specific bad parameter . SO it should be one test per field in error . There is a good example of doing this with parameters in messages_get on master